src/controller/CommissioningDelegate.h

```diff
@@ -266,6 +266,15 @@ struct AttestationResponse
     ByteSpan signature;
 };
 
+struct OpCertResponse
+{
+    OpCertResponse(ByteSpan newAttestationElements, ByteSpan newSignature) :
+        attestationElements(newAttestationElements), signature(newSignature)
+    {}
+    ByteSpan attestationElements;
```

Comment:
Suggest renaming this to field nocsrElements, since that's what they are.

Also, the name of it should be NOCSRResponse